src/app.tsx

```diff
@@ -15,25 +15,25 @@ import './global.scss';
 
 const root = document.getElementById('flow-editor');
 
-// if (__flow_editor_config__.hasOwnProperty('path')) {
-//     __webpack_public_path__ = __flow_editor_config__.path;
-// }
+if (process.env.NODE_ENV === 'production') {
```

Comment:
keeping hot-loading code out of production bundles (even if it won't run in production, it adds to the bundle's size)